src/mender-update/cli/cli.cpp

```diff
@@ -49,6 +49,11 @@ const conf::CliCommand cmd_daemon {
 const conf::CliCommand cmd_install {
 	.name = "install",
 	.description = "Mender Artifact to install - local file or a URL",
+	.argument =
+		{
+			.name = "manifest",
+			.mandatory = false,
```

Comment:
As I indirectly wrote [here](https://github.com/mendersoftware/mender/pull/1601#discussion_r1535637076) and directly wrote [here](https://github.com/mendersoftware/mender/pull/1601#discussion_r1535638579), for `mender-update` the argument is "artifact" and it is mandatory.

Reply:
Oh sorry, fixed it now